CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 ## 0.6.0-dev
   **BUG FIXES**
+  - [#123](https://github.com/Datatamer/unify-client-python/issues/123) Fix base_path bug for a custom api endpoint
+
+## 0.5.0-dev
```

Comment:
On `Datatamer/master` we've already changed this to `0.5.0` (removing the `-dev` suffix). Likely you got this coming back in because you haven't updated your local `master` branch and then `rebase`'d your feature branch on that. Ask @Nelson-Andrade for help on this if you need it.

In other words, this `0.5.0-dev` change should not be part of your PR.

Reply:
done

Reply:
@CaspianA1 I'm unresolving this thread until the code change is reflected in this PR. In general, I recommend you refrain from resolving Github comments until after the relevant code is changed/updated in the PR, that way reviewers can keep track of which things still need to be done.